### ISSUE TITLE
[Bug]: NC 28.0.2 Context menu in "files" app does not work at all

### ISSUE THREAD

**Issue by arnowelzel** [nextcloud/server#43300]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When clicking a file in the "files" app, the context menu opens and the closes immediately again, so you can not use it.

### Steps to reproduce

1. Open a the "files" app
2. Click a file entry with the right mouse button
3. See that the context menu opens and closes again

### Expected behavior

The context menu works as expected (as in NC 28.0.1).

### Installation method

Community Manual installation with Archive

### Nextcloud Server version

28

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.3

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated from a MINOR version (ex. 22.1 to 22.2)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by mehlkopf** [comment]
Having the exact same problem, independent of browser (e.g. Safari or Brave), on a Mac. This 28 version cycle is frustrating …………

**Comment by claushviid** [comment]
Having the exact same problem too.

**Comment by poiNt3D** [comment]
I confirm this.
It basically makes right-click menu useless.

**Comment by r4lix** [comment]
Same here, I upgraded from 26 to 28, I should have stayed with 26.

**Comment by nodiscc** [comment]
Confirming this bug on 28.0.2.

Right-click menu is unusable after scrolling down the file list.

**Steps to reproduce:**
- upload a few dozen files in a directory
- scroll down the file list
- right-click a file

**Expected:**
- the menu stays open and allows me to select an action such as `Rename`, `Download`, etc.

**What happens instead:**
- the menu closes immediately and the view scrolls back all the way up the file list

**Workaround:**
- this only happens in the default list view, switching to grid/thumbnails view allows working around the problem (but is not practical)

**Comment by MrRinkana** [comment]
For me the right click menu stays, but the page is instantly scrolled up.

I'm not sure how the right click menu is supposed be placed, but that looks a bit off aswell (quite far from both mouse and directory, independent of whether the scroll jumped or not).

I'm on 28.0.2

Similar jump in window position at https://github.com/nextcloud/notes/issues/1227

**Comment by MrRinkana** [comment]
Some additional findings:

The scroll jump only occurs when right-clicking the top most visible folder/file or the lowest one while in listview. The lowest one always scrolls (if you can scroll) but sometimes the lowest visible also can trigger the jump, seems to mostly trigger on the bottomost few.

In gridview it seems less predictable, sometimes right-clicking the topmost visible makes it jump down for example. Clicking the bottommost can trigger the jump, but only if it is alone on the last row (?).

I mentioned that the rightclick menu opens in only one location but that is not completely true, as clicking the topmost row produces one location, the rest another (to two locations in total). Sometimes reloading the page fixes the weird location problem, albeit now the options are highlighted as if some accessibility option got turned on.

In essence; things are weird :sweat_smile:

**Comment by wargreen** [comment]
> Similar jump in scroll position at [nextcloud/notes#1227](https://github.com/nextcloud/notes/issues/1227)

Same thing happen with the pdf viewer.

**Comment by theosche** [comment]
> Having the exact same problem, independent of browser (e.g. Safari or Brave), on a Mac. This 28 version cycle is frustrating …………

I also have this problem with 28.0.3 using Firefox (123.0) and Safari (16.5.1), but the menu works on Chrome (122.0.6261.94).

**Comment by refda** [comment]
I was testing the right click menu again with 28.0.3 in Firefox and Chrome: Both are not working when it comes to the position, please see https://github.com/nextcloud/server/issues/43842

**Comment by MrRinkana** [comment]
Maybe https://github.com/nextcloud/notes/issues/1227 should be closed aswell, since it seems fixed and i suspect it was fixed by the same changes

**Comment by Pytal** [comment]
The fix seems unrelated as https://github.com/nextcloud/server/pull/44139 only touches files code which does not interfere with notes code

If the issue cannot be reproduced anymore though, feel free to close it @MrRinkana

**Comment by MrRinkana** [comment]
You're right, it was fixed by https://github.com/nextcloud/text/pull/5281

I cannot close the issue, but I made a comment asking op to close if it no longer reproduces
